packages/@coorpacademy-components/src/molecule/dashboard/cards-list/index.js

```diff
@@ -309,31 +309,31 @@ class CardsList extends React.PureComponent {
       map(([key, card]) => {
         return (
           <div className={style.card} key={key}>
-            <Card {...card} dataName={`${dataName}-${key}`} />
+            {card.type === 'learningPriority' ? (
```

Comment:
this cause error in some fixtures: https://6622682e70ecd8ecf5027fa1-udtfhysbfc.chromatic.com/?path=/story/molecule-dashboard-cardslist--performance

Reply:
ah yes good point, I fix that!